doc/src/manual/environment-variables.md

```diff
@@ -406,6 +406,19 @@ should have at the terminal.
 The formatting `Base.answer_color()` (default: normal, `"\033[0m"`) that output
 should have at the terminal.
 
+### [`NO_COLOR`](@id NO_COLOR)
+
+When this variable is present and not an empty string (regardless of its value) then colored
+text will be disabled on the REPL. Can be overridden with the flag `--color=yes` or with the
+environmental variable [`FORCE_COLOR`](@ref FORCE_COLOR). This environmental variable is
+[commonly recognized by command-line applications](https://no-color.org/).
+
+### [`FORCE_COLOR`](@id FORCE_COLOR)
+
+When this variable is present and not an empty string (regardless of its value) then
+colored text will be enabled on the REPL. Can be overridden with the flag `--color=no`. This
+environmental variable is [commonly recognized by command-line applications](https://force-color.org/).
```

Comment:
```suggestion
environment variable is [commonly recognized by command-line applications](https://force-color.org/).
```

Reply:
I am not sure I'd agree with the claim that this is "commonly" recognized, the list on their website is rather short. I am also not sure if it is helpful in any way to include these claims and also the links to no-color.org and force-color.org in here. But that's just my two cents, dunno *shrug*.

In the end it is a trivial change either way, I'd be happy to merge this either way.

Reply:
The reason I wanted to include these links at all is that if someone sets `NO_COLOR=1` when running Julia these options will be picked up by processes created within Julia as well. It also lets the reader know why we didn't name these `JULIA_NO_COLOR` and `JULIA_FORCE_COLOR`.